django_afip/factories.py

```diff
@@ -120,6 +121,12 @@ class Meta:
     sales_terms = "Credit Card"
 
 
+class PointOfSalesFactoryCaea(PointOfSalesFactory):
+
+    number = Sequence(lambda n: n + 1)
```

Comment:
Number here must be `4`.

Reply:
Fixed and tested, everything run perfectly